vnext/ReactUWP/Modules/Animated/SpringAnimationDriver.h

```diff
@@ -16,19 +16,20 @@ class SpringAnimationDriver : public AnimationDriver {
       int64_t animatedValueTag,
       const Callback &endCallback,
       const folly::dynamic &config,
-      const std::shared_ptr<NativeAnimatedNodeManager> &manager);
+      const std::shared_ptr<NativeAnimatedNodeManager> &manager,
+      const folly::dynamic &dynamicToValues = folly::dynamic::array());
 
   std::tuple<winrt::CompositionAnimation, winrt::CompositionScopedBatch>
   MakeAnimation(const folly::dynamic &config) override;
 
+  double ToValue() override;
+
  private:
   std::tuple<float, double> GetValueAndVelocityForTime(
       double time,
       double startValue);
-  bool IsAtRest(
-      double currentVelocity,
-      double currentPosition,
-      double startPosition);
+  bool
+  IsAtRest(double currentVelocity, double currentPosition, double endValue);
```

Comment:
What was wrong about how it was formatted before?

Reply:
clang changed it with the name change <shrug>

---
In reply to: [324896812](https://github.com/microsoft/react-native-windows/pull/3041#discussion_r324896812) [](ancestors = 324896812)